src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -525,9 +525,8 @@ public override IntermediateNode Visit(DottedNameNode node, IRTranslatorContext
 
                     if (DType.Color.Accepts(resultType))
                     {
-                        Contracts.Assert(value is uint);
-                        result = new ColorLiteralNode(context.GetIRContext(node), (uint)value);
-                    }
+                        result = new ColorLiteralNode(context.GetIRContext(node), System.Drawing.Color.FromName(node.Right.Name.ToString()));
```

Comment:
The keys are already mapped to specific colors in the enum definition, which is the uint value that was used here previously. I'd suggest using that value rather than relying on c#'s mapping

Reply:
our colorvalue function required input is 'Color' type, not uint. we still need to convert from uint to color, how we do that? system.drawing only convert from int to color.

Reply:
See LibraryUnary::OptionSetValueToColor.